limited/ravenkeeper

```diff
@@ -1,13 +1,13 @@
 **Ravenkeeper** | Townsfolk Power | Limited
 __Basics__
-Each night, the Ravenkeeper may assign one of its ravens to a player. The Ravenkeeper will learn if an ability is used on of the players with a raven.
+Each night, the Ravenkeeper may assign one of its ravens to a player. The Ravenkeeper will learn if an ability is used on any of the players with a raven.
 __Details__
 The Ravenkeeper starts out with 1 raven for every 5 players in the game (rounded down). Each night, the Ravenkeeper may assign one raven to another player. Assigning a raven is an *immediate effect*.
-The first time an ability is used on a player with a raven, the raven heads back to the Ravenkeeper and lets the Ravenkeeper learn the type of ability that was used and which player it was used on. Ravens can usually only report on one ability, except when several abilities are used at the same time at the end of a phase (in this case the Ravenkeeper will learn all the ability types, but not duplicates). 
-The Ravenkeeper is affected by *redirections*, however the Ravenkeeper will know a raven got redirected when it returns and reports on an ability.
+The first time an ability is used on a player with a raven, the raven returns to the Ravenkeeper and lets the Ravenkeeper learn the type of ability that was used and which player it was used on. Ravens can usually only report on one ability, except when several abilities are used at the same time. In this case, the Ravenkeeper will learn all the ability types except duplicates. 
```

Comment:
"Ravens can usually only report on one ability, except when several abilities are used at the same time." - at the same time just doesn't exist. Do we change this?  This doesn't seem to be implemented either

Reply:
yeah it's just 1 ability